src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -130,45 +145,59 @@ const StreamGroup = createReactClass({
     const {organization, query, selection} = this.props;
     const {data} = this.state;
 
-    // TODO: @taylangocmen when there is no discover query open events page
-
-    const {period, start, end} = selection.datetime || {};
+    // when there is no discover feature open events page
+    const hasDiscoverQuery = organization.features.includes('discover-basic');
 
-    const discoveryQueryTerms = [];
+    const queryTerms = [];
 
     if (filtered && query) {
       const queryObj = queryToObj(query);
       for (const queryTag in queryObj)
-        if (!['is', '__text'].includes(queryTag)) {
+        if (!DiscoveryExclusionFields.includes(queryTag)) {
           const queryVal = queryObj[queryTag].includes(' ')
             ? `"${queryObj[queryTag]}"`
             : queryObj[queryTag];
-          discoveryQueryTerms.push(`${queryTag}:${queryVal}`);
+          queryTerms.push(`${queryTag}:${queryVal}`);
         }
+
+      if (!hasDiscoverQuery && queryObj.__text) queryTerms.push(queryObj.__text);
```

Comment:
we tend to still use brackets for single line statements

Reply:
Can someone look into adding this as a lint rule?

Reply:
Thanks